bindings/cs/MongoDB.Libmongocrypt.Test/SigningRSAESPKCSCallbackTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * Copyright 2020–present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Text;
+using FluentAssertions;
+using Xunit;
+
+namespace MongoDB.Libmongocrypt.Test
+{
+    public class SigningRSAESPKCSCallbackTests
+    {
+        private static string DataToSign =  "data to sign";
+
+        private static string PrivateKey = Environment.GetEnvironmentVariable("FLE_GCP_RSA_SIGNING_PKCS8_PRIVATE_KEY") ?? throw new ArgumentNullException("FLE_GCP_RSA_SIGNING_PKCS8_PRIVATE_KEY has not been specified.");
```

Comment:
Note, the private key from the "Drivers guide to support Azure and GCP in CSFLE" was generated for the purpose of testing signatures, so it does not need to be omitted from source. But is probably better to do so anyway to avoid the risk of using the wrong key.

Reply:
I wasn't sure that it's safe to leave this key in public access. But if so, it's better to do it to avoid unnecessary configuring for env variables that won't be used in any other place. Thanks for clarifying it.